index.js

```diff
@@ -22,10 +22,9 @@ function ElasticProvider(config) {
 
   if (!config.elasticCallConcurrency) config.elasticCallConcurrency = 20;
 
-  if (!config.bulkBachSizes) config.bulkBachSizes  = 1000;
-
-  if (!config.bulkMaxSize) config.bulkMaxSize  = 100000;
+  if (!config.bulkBachSizes) config.bulkBachSizes = 1000;
```

Comment:
is it still possible to fix spelling of this field name? Should be bulkBatchSizes (with a t)

Reply:
done